frontend/ui/widget/spinwidget.lua

```diff
@@ -204,7 +188,7 @@ function SpinWidget:update()
     table.insert(vgroup, CenterContainer:new{
         dimen = Geom:new{
             w = self.width,
-            h = value_group:getSize().h + math.floor(self.screen_height * 0.1),
+            h = value_group:getSize().h + math.floor(math.max(self.screen_width, self.screen_height) * 0.06),
```

Comment:
This 0.06 also feels random :) But not sure what this CenterContainer contains and why 6% of max screen dimension is right for it - so dunno if it can be a fixed Size.something.

Reply:
> feels random

Just aesthetically, as in the screenshots.

Reply:
What does this makes ? I guess it adds some vertical padding/span above and below the natural height of the NumberPickerWidget, by centering it in this added height, right ?
Can't you use `h = value_group:getSize().h + 2*Size.padding.large` (or `Size.span.vertical_large` or a new `Size.span.vertical_verylarge`) ?

Reply:
Although using Size will make this span/padding larger when using a high DPI.
But may be that's what should happen with a high DPI ? @Frenzie : thoughts ?

Reply:
<kbd>![5](https://user-images.githubusercontent.com/62179190/133431382-37354d33-11e3-49b1-a6a9-79eef5f552d6.png)</kbd>

Reply:
Dunno what you used, but that looks ok to me ?

Reply:
We lock the width, while height is combined with the heights of elements, so proportions change.

Reply:
<kbd>![7](https://user-images.githubusercontent.com/62179190/133432670-b5861d93-e0b4-47ea-83bb-8b544f284f31.png)</kbd>

Reply:
> Dunno what you used

The latest 2 commits.

Reply:
Please don't just say "160:", also say what you think of the result - because sometimes, I can't guess if you're happy or not :)
The last screenshot feels a bit too packed at first sight - but well, the different spacings looks similar all around (between title bar and first button's top, between first button's bottom and "22.0), so it may be right?

Reply:
Other dialogs with fixed width (InputDialog, ButtonDialog) also look a little bit flattened at lower dpi, so I think it's okay.